procrustes/utils.py

```diff
@@ -336,3 +339,55 @@ def _check_arraytypes(*args):
         raise TypeError("Matrix inputs must be NumPy arrays")
     if any(x.ndim != 2 for x in args):
         raise TypeError("Matrix inputs must be 2-dimensional arrays")
+
+
+class OptResult(dict):
+    r"""Represents the Procrustes optimization result.
+
+    Attributes
+    ----------
+    new_a : ndarray
+        The translated/scaled numpy ndarray :math:`\mathbf{A}`.
+    new_b : ndarray
+        The translated/scaled numpy ndarray :math:`\mathbf{B}`.
+    array_u : ndarray
+        The right hand side optimum transformation matrix.
+    array_p : ndarray
+        The left hand side transformation matrix for two-sided Procrustes problem with
+        two transformation.
+    array_q : ndarray
+        The right hand side transformation matrix for two-sided Procrustes problem with
+        two transformation.
+    e_opt : float
+        Two-sided permutation Procrustes error.
+
+    Notes
+    -----
+    Some of additional attributes are not listed above can be foundt at specific Procrustes
```

Comment:
Can you give a case where additional arguments might be found that are not listed above?

Reply:
We don't have extra arguments right now. I was thinking to add some message information, such as how many interations have been run, in the object. But that's the future plan. So it's safe to remove this confusing `Notes`.